apiBase/errors/NotImplementedError.js

```diff
@@ -1,5 +1,3 @@
-import FakeError from './FakeError';
```

Comment:
After the issue with `BadCaptchaError`, should we make this extend `FakeError` as well so client's can use `instanceof`?